tests/resources/test_comment.py

```diff
@@ -9,9 +10,10 @@ def setUp(self):
         self.issue_1_key = self.test_manager.project_b_issue1
         self.issue_2_key = self.test_manager.project_b_issue2
         self.issue_3_key = self.test_manager.project_b_issue3
+        self.issue_4_key = self.test_manager.project_b_issue4
 
     def tearDown(self) -> None:
-        for issue in [self.issue_1_key, self.issue_2_key, self.issue_3_key]:
+        for issue in [self.issue_1_key, self.issue_2_key, self.issue_3_key, self.issue_4_key]:
```

Comment:
Please reuse an existing issue for your test.

This issue key does not exist, all issues are created
https://github.com/pycontribs/jira/blob/main/tests%2Fconftest.py#L317

Reply:
sorry @adehad - I considered the issue creation more generic.
I hope f206010 fixes my mistake ...